### PR TITLE
add refresh-design divider secondary option

### LINKED ISSUE #516: Add new divider option
Add divider with `secondary` option. The divider for refreshed design looks like:

Screenshot 2023-07-26 at 21 50 32

### PR DESCRIPTION
**Pull request by Acidham**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #516 

Test URLs:
- Before: https://main--mammotome--hlxsites.hlx.page/us/en/experimentation/hero-carousel
- After: https://516-add-new-divider-option--mammotome--hlxsites.hlx.page/us/en/experimentation/hero-carousel

### CODE REVIEW

File: docs/section-metadata.md [hunk 1]
```diff
@@ -231,21 +231,27 @@ Narrow style puts the content in a section that is fixed to **610px**.
...
 **_Example_**
 
 
-| Section Metadata |       |
-| ---------------- | ----- |
-| divider          | after |
-
+| Section Metadata |                  |
+| ---------------- |------------------|
+| divider          | primary or _empty_ |
```
Comment (dzehnder) [inline docs/section-metadata.md:244]: `secondary or _empty_`?
As primary seems to be the default option when empty

File: docs/section-metadata.md [hunk 2]
```diff
@@ -231,21 +231,27 @@ Narrow style puts the content in a section that is fixed to **610px**.
 Adds a divider before or after the section.
```
Comment (dzehnder) [inline docs/section-metadata.md:231]: Description probably needs to be updated as well, as there does not seem to be an option for `before` and `after` anymore.

File: styles/styles.css [hunk 3]
```diff
@@ -1010,14 +1012,23 @@ main .section.our-history .year.highlight .icon-calendar {
...
   margin: calc(26px + 10px + 3%) auto auto;
 }
 
+.section hr.divider.secondary {
+  border: 0;
+  height: 4px;
+  background-color: var(--mt-global-horizontal-line-color-blue);
+  width: 34px;
+  margin: calc(26px + 10px + 3%) auto auto;
+}
```
Comment (dzehnder) [inline styles/styles.css:1029]: CSS properties which are the same for primary and secondary could be consolidated into a common selector:

```
.section hr.divider {
  border: 0;
  height: 4px;
  margin: calc(26px + 10px + 3%) auto auto;
}
```

File: docs/section-metadata.md (hunk 1 above)
Comment (Acidham) [reply docs/section-metadata.md:244]: It is the example table. Secondary example comes next

**Review by dzehnder — APPROVED** [review]

**Pull request merged**